sdk/requests/index.js

```diff
@@ -20,16 +21,23 @@ const makeRequest = async type => {
         Linking.openURL(loginEndpoint(parameters.clientId))
       );
     }
-    case REQUEST_TYPES.GET_TOKEN: {
+    case REQUEST_TYPES.GET_TOKEN:
+    case REQUEST_TYPES.GET_REFRESH_TOKEN: {
```

Comment:
```suggestion
    case REQUEST_TYPES.GET_TOKEN || REQUEST_TYPES.GET_REFRESH_TOKEN: {
```
Creo que lo podes poner así, me parece que queda mas lindo, pero no estoy seguro si funciona